src/amo/sagas/reviews.js

```diff
@@ -297,7 +297,7 @@ function* deleteAddonReview({
       reviewId,
     });
 
-    yield put(unloadAddonReviews({ reviewId }));
+    yield put(unloadAddonReviews({ reviewId: isReplyToReviewId || reviewId }));
```

Comment:
It would be nice if the reducer for `unloadAddonReviews` also unloaded `state.view[reviewId]`. I don't see any bugs because of it but it would leave the UI a bit fragile since `view` is used to control things like showing and hiding the reply / edit forms.

https://github.com/mozilla/addons-frontend/blob/82e55db015d072c1baa8fe59265fcfd337c66966/src/amo/reducers/reviews.js#L97

Clearing `state.view[reviewId]` is also important since it functions independently of `state.byId`.

Reply:
I was actually wondering about this when I wrote `unloadAddonReviews` but wasn't sure if it was necessary. Thanks for letting me know. 👍